commands.js

```diff
@@ -1465,6 +1465,26 @@ var commands = exports.commands = {
 	},
 	forcerenamehelp: ["/forcerename OR /fr [username], [reason] - Forcibly change a user's name and shows them the [reason]. Requires: % @ & ~"],
 
+	hidetext: function (target, room, user) {
+		if (!target) return this.parse('/help hidetext');
+
+		var reason = this.splitTarget(target);
+		var targetUser = this.targetUser;
+		var name = this.targetUsername;
```

Comment:
Your `/hidetext` command, if implemented this way, only works for the user's most recent username. We should probably support older usernames in case the user changes their mind.